scormxblock/scormxblock.py

```diff
@@ -138,7 +138,7 @@ def updoad_all_content(temp_directory, fs):
     """
     This standalone function handles the bulk upload of unzipped content.
     """
-    if not settings.DJFS.get('type', 'osfs') == "s3fs":
+    if settings.DJFS.get('type', 'osfs') != "s3fs":
```

Comment:
I think `not` it's more explicit.

Reply:
I changed it because I think is more immediate to notice what the line does with de !=. It happened to me that I didn't understand the line because i was looking only at ==. But it can be me, what do you think?